versions.gradle

```diff
@@ -12,7 +12,6 @@ ext.androidVersions = androidVersions
 
 ext.deps = [:]
```

Comment:
This file should be removed, I assume this is a WIP branch, am I right? Add WIP to the branch name

Reply:
I suppose that this file will be removed after migrating all the dependencies